json_annotation/lib/src/json_serializable.dart

```diff
@@ -22,7 +46,10 @@ class JsonSerializable {
   /// `null` runtime validation if it's critical.
   final bool nullable;
 
-  // TODO(kevmoo): document the constructor
+  /// Creates a [JsonSerializable] instance.
+  ///
+  /// For details on each parameter, see the documentation for the corresponding
```

Comment:
[nit] do we think this is something that we need to state explicitly? I don't think I've ever hesitated to jump to field doc comments.